<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TopTitle</class>
 <widget class="QWidget" name="TopTitle">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>80</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>80</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 85, 255);</string>
  </property>
  <widget class="QWidget" name="left_wgt" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>111</width>
     <height>80</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>111</width>
     <height>80</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>111</width>
     <height>16777215</height>
    </size>
   </property>
   <widget class="QPushButton" name="pb_min_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>64</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>64</width>
      <height>64</height>
     </size>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>当前用户</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color:transparent;
	background-image: url(:/EquivalentSoft/images/logo4.png);
    border:0px;
}

</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="center_wgt" native="true">
   <property name="geometry">
    <rect>
     <x>111</x>
     <y>0</y>
     <width>662</width>
     <height>80</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>662</width>
     <height>80</height>
    </size>
   </property>
   <widget class="QLabel" name="lb_title">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>20</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>SimSun-ExtB</family>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="text">
     <string>箭上控制等效器YUI</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="right_wgt" native="true">
   <property name="geometry">
    <rect>
     <x>773</x>
     <y>0</y>
     <width>251</width>
     <height>80</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>251</width>
     <height>80</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>251</width>
     <height>16777215</height>
    </size>
   </property>
   <widget class="QPushButton" name="pb_resize">
    <property name="geometry">
     <rect>
      <x>185</x>
      <y>10</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>最大化</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:hover{
	background-color:transparent;
	background-image:url(:/EquivalentSoft/images/放大-悬浮.png);
    border:0px;}
QPushButton:pressed{
	background-color:transparent; 
    background-image: url(:/EquivalentSoft/images/放大-点击.png);
    border:0px;}
QPushButton {
	background-color:transparent;
	background-image: url(:/EquivalentSoft/images/放大-默认.png);
    border:0px;
}

</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_min">
    <property name="geometry">
     <rect>
      <x>155</x>
      <y>10</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>最小化</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:hover{
	background-color:transparent;
	background-image:url(:/EquivalentSoft/images/最小化-悬浮.png);
    border:0px;}
QPushButton:pressed{
	background-color:transparent; 
    background-image: url(:/EquivalentSoft/images/最小化-点击.png);
    border:0px;}
QPushButton {
	background-color:transparent;
	background-image: url(:/EquivalentSoft/images/最小化-默认.png);
    border:0px;
}

</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lb_user">
    <property name="geometry">
     <rect>
      <x>85</x>
      <y>25</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(15, 11, 49);</string>
    </property>
    <property name="text">
     <string>admin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_close">
    <property name="geometry">
     <rect>
      <x>215</x>
      <y>10</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>关闭</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:hover{
	background-color:transparent;
	background-image: url(:/EquivalentSoft/images/关闭-悬浮.png);
    border:0px;}
QPushButton:pressed{
	background-color:transparent; 
	background-image: url(:/EquivalentSoft/images/关闭-点击.png);
    border:0px;}
QPushButton {
	background-color:transparent;
	background-image: url(:/EquivalentSoft/images/关闭-默认.png);
    border:0px;
}

</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_min_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>当前用户</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color:transparent;
	background-image: url(:/EquivalentSoft/images/用户1.png);
    border:0px;
}

</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>pb_resize</zorder>
   <zorder>pb_min</zorder>
   <zorder>lb_user</zorder>
   <zorder>pb_close</zorder>
   <zorder>pb_min_2</zorder>
   <zorder>center_wgt</zorder>
  </widget>
 </widget>
 <resources>
  <include location="equivalentsoft.qrc"/>
 </resources>
 <connections/>
</ui>
